Pop_Quiz.txt

```diff
@@ -10,6 +10,7 @@ git add file_a file_b file_c
 4. Explain what mv does and give an example of how to use it.
 
 5. I made committed changes to my main branch by mistake. Do I have to redo those changes on a new branch?
+checkout to branch then then commit.
 
 6. If not, what should I do?
 
```

Comment:
line 13 is the answer to 6 line 13 should say 'no'

Reply:
Changes made and updated.